skcosmo/metrics/__init__.py

```diff
@@ -0,0 +1,21 @@
+from ._reconstruction_measures import (
+    pointwise_global_reconstruction_error,
+    global_reconstruction_error,
+    pointwise_global_reconstruction_distortion,
+    global_reconstruction_distortion,
+    pointwise_local_reconstruction_error,
+    local_reconstruction_error,
+    check_global_reconstruction_measures_input,
+    check_local_reconstruction_measures_input,
+)
+
+__all__ = [
```

Comment:
I'm not sure I see the point of specifying `__all__` here (and everywhere else). Is it to prevent importing `skcosmo.metrics._reconstruction_measures`?

Reply:
I think I had some issue in the early version of skcosmo without `__all__` to import skcosmo or I wanted to hide some functions, but I dont remember anymore what the reason for this was.
I removed it now everywhere.

Reply:
ah yes now I remember, the lint tests failed^^ I add the `__all__` after you had a chance to check out the error

Reply:
Ok, I see the error and I disagree with pylint, but I think is it better to leave any change here for a separate PR =)